src/explore-education-statistics-admin/src/pages/release/edit-release/data/ReleaseDataPage.tsx

```diff
@@ -3,19 +3,69 @@ import ManageReleaseContext, {
 } from '@admin/pages/release/ManageReleaseContext';
 import Tabs from '@common/components/Tabs';
 import TabsSection from '@common/components/TabsSection';
-import React, { useContext } from 'react';
+import React, { useCallback, useContext, useEffect, useState } from 'react';
+import footnotesService from '@admin/services/release/edit-release/footnotes/service';
+import permissionService from '@admin/services/permissions/service';
+import { generateFootnoteMetaMap } from '@admin/services/release/edit-release/footnotes/util';
+import {
+  Footnote,
+  FootnoteMeta,
+  FootnoteMetaGetters,
+} from '@admin/services/release/edit-release/footnotes/types';
+import { ErrorControlProps } from '@admin/validation/withErrorControl';
+import ReleaseFootnotesSection from '@admin/pages/release/edit-release/data/ReleaseFootnotesSection';
 import ReleaseDataUploadsSection from './ReleaseDataUploadsSection';
 import ReleaseFileUploadsSection from './ReleaseFileUploadsSection';
-import ReleaseFootnotesSection from './ReleaseFootnotesSection';
 
-const ReleaseDataPage = () => {
+export interface FootnotesData {
+  loading: boolean;
```

Comment:
Shouldn't need to have a `loading` prop in the data as it looks like `loading` will only ever be false.

If there is actually a loading state that gets toggled, this should be set in some other state variable and passed to `ReleaseFootnotesSection` separately.